### ISSUE TITLE
iOS: Implement Menu (Navigation)

### ISSUE THREAD

**Issue by Nicolas-M** [Bathlamos/RTDC#103]
Depends on #102.

Once a user is able to login, they should be able to switch between different features in the app: Capacity Overview, Action Plan, Communication Hub, Manage Users, Manage Units. 

Although we're using a drawer menu in Android, I would highly recommend using a Tab Bar for navigation in iOS since they are more common.

**Comment by Nicolas-M** [comment]
iOS is now out of scope for this project. Closing issue.
